test/optim/test_initializers.py

```diff
@@ -241,3 +248,48 @@ def test_gen_one_shot_kg_initial_conditions(self):
                     )
                 )
                 self.assertTrue(torch.all(ics[..., -n_value:, :] == 1))
+
+
+class TestGenValueFunctionInitialConditions(BotorchTestCase):
+    def test_gen_value_function_initial_conditions(self):
+        num_fantasies = 8
+        num_solutions = 3
+        num_restarts = 4
+        raw_samples = 16
+        n_train = 4
```

Comment:
let's try to get these numbers down to the minimum, every ms counts for the tests
```suggestion
        num_fantasies = 2
        num_solutions = 3
        num_restarts = 4
        raw_samples = 5
        n_train = 6
```